model/time.go

```diff
@@ -163,9 +163,21 @@ func (t *Time) UnmarshalJSON(b []byte) error {
 // This type should not propagate beyond the scope of input/output processing.
 type Duration time.Duration
 
+// Set implements pflag/flag.Value
+func (d *Duration) Set(s string) error {
+	var err error
+	*d, err = ParseDuration(s)
+	return err
+}
+
+// Type implements pflag.Value
+func (d *Duration) Type() string {
+	return "model.Duration"
```

Comment:
@Gouthamve I wasn't aware this is the value which will be printed in the help page / usage. I don't think it makes sense to use our internal data model here. Probably just `duration`?